src/ui/Alert/Alert.tsx

```diff
@@ -0,0 +1,122 @@
+import { cva, VariantProps } from 'cva'
+import * as React from 'react'
+
+import { cn } from 'shared/utils/cn'
+import Icon from 'ui/Icon'
+
+const alertVariants = cva('relative w-full border-l-4 p-4', {
+  variants: {
+    variant: {
+      error: 'border-ds-primary-red bg-error-100',
+      info: 'border-ds-blue-darker bg-ds-blue-nonary',
+      success: 'border-green-500 bg-green-100',
+      warning: 'border-orange-500 bg-orange-100',
+    },
+  },
+  defaultVariants: {
+    variant: 'info',
+  },
+})
+
+export function variantToIcon(variant?: string | null) {
+  let classname = 'float-left -mt-1 mr-2 align-middle'
```

Comment:
I think this is a typo
```suggestion
  let classname = 'float-left mt-1 mr-2 align-middle'
```

Reply:
was intended actually! That's a -1 pixel, basically move the image slightly above so the middle is more inline with the middle of the text

Reply:
duuude I'm learning so much tailwind from this one PR this is wild

Reply:
This is actually my first gig using tailwind in production too LOL
Having to google a lot of my CSS -> Tailwind conversions

Reply:
What if I told you this isn't my first gig using tailwind in production 🥲 

Keep learning stay humble 💯 😂

Reply:
@ajay-sentry read the tailwind docs ... they are amazing